assets/js/common/SuseManagerSettingsDialog/SuseManagerSettingsModal.stories.jsx

```diff
@@ -0,0 +1,85 @@
+import SuseManagerSettingsModal from './SuseManagerSettingsModal';
+
+export default {
+  title: 'Components/SuseManagerSettingsModal',
+  component: SuseManagerSettingsModal,
+  argTypes: {
+    open: {
+      description: 'Whether the dialog is open or not',
+      control: {
+        type: 'boolean',
+      },
+    },
+    loading: {
+      description: 'Whether the settings are loading or submitting',
+      control: {
+        type: 'boolean',
+      },
+    },
+    initialUsername: {
+      description: 'Initial SUSE Manager username',
+      control: {
+        type: 'string',
+      },
+    },
+    initialUrl: {
+      description: 'Initial SUSE Manager URL',
+      control: {
+        type: 'string',
+      },
+    },
+    certUploadDate: {
+      description: 'Certificate upload date',
+      control: {
+        type: 'date',
+      },
+    },
+    errors: {
+      description: 'OpenAPI errors coming from backend validation',
+    },
+  },
+};
+
+export const Default = {
+  args: {
+    open: false,
```

Comment:
What about a trigger in the stories to open the dialog? Or alternatively `open: true` to all of them, if it does not become too invasive.